### PR TITLE
p2-b5-r8 Paris

### PR DESCRIPTION
**Pull request by GrantSchreib**

### CODE REVIEW

File: bHY6yh-d38247/cod-bxxqqq_bHY6yh-d38247.xml [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+         <publicationStmt>
+            <authority>SCTA</authority>
+            <availability status="free">
+               <p>Published under a <ref target="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</ref>
+               </p>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <listWit>
+               <witness xml:id="V" n="cod-pu6y7d">Venice 1505</witness>
```
Comment (jeffreycwitt) [inline bHY6yh-d38247/cod-bxxqqq_bHY6yh-d38247.xml:33]: doesn't look like the witness declaration got changed.

File: bHY6yh-d38247/cod-bxxqqq_bHY6yh-d38247.xml [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+            <p xml:id="bHY6yh-d38247-d1e114">
+              <lb ed="#B"/>De credito et debito Rubrica octaua
+            </p>
+            <p xml:id="bHY6yh-d38247-d1e119">
+              <lb ed="#B"/><ref xml:id="bHY6yh-d38247-Rd1e115">PSalmista 36 pslamus</ref> <quote xml:id="bHY6yh-d38247-Qd1e123">Mutuabi<lb ed="#B" break="no"/>t  ur 
+              peccatori et non soluet</quote> 
+            </p>
+            <p xml:id="bHY6yh-d38247-d1e131">
+              <lb ed="#B"/><ref xml:id="bHY6yh-d38247-Rd1e129">SAlomon in prouerbiis capitulo 6</ref> <quote xml:id="bHY6yh-d38247-Qd1e135">Si spon<lb ed="#B" break="no"/>s  ponderis 
+              pro amico tuo defix<lb ed="#B" break="no"/>i  sti
```
Comment (jeffreycwitt) [inline bHY6yh-d38247/cod-bxxqqq_bHY6yh-d38247.xml:72]: I think something went wrong with your line break search and replace.

**Review by jeffreycwitt — CHANGES_REQUESTED** [review]
I think something went wrong with your line break search and replace. Please review throughout

File: bHY6yh-d38247/cod-bxxqqq_bHY6yh-d38247.xml [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+              <lb ed="#B"/>in ewangelio capitulo 11</ref> <quote xml:id="bHY6yh-d38247-Qd1e195"></quote>
+            </p>
+            <p xml:id="bHY6yh-d38247-d1e201">
+               ¶ <ref xml:id="bHY6yh-d38247-Rd1e207">Idem dicit lucas in ewangelio 
+              <lb ed="#B"/>capitulo 20</ref> <quote xml:id="bHY6yh-d38247-Qd1e203"> </quote>
+            </p>
+            <p xml:id="bHY6yh-d38247-d1e215">
+              <lb ed="#B"/><ref xml:id="bHY6yh-d38247-Rd1e226">PAulus<!--line jumps up--> apostolus epistola ad romanos<!--line jumps down--> capitulo 13</ref>  ¶ <quote xml:id="bHY6yh-d38247-Qd1e219">Reddite omnibus 
+              <lb ed="#B"/>debita cui tribulum tributum 
+              <lb ed="#B"/>cui vectigal vectigal. cui hono<lb ed="#B" break="no"/>r  em
```
Comment (jeffreycwitt) [inline bHY6yh-d38247/cod-bxxqqq_bHY6yh-d38247.xml:113]: there's still a problem with the linebreaks here and throughout

**Review by jeffreycwitt — CHANGES_REQUESTED** [review]

File: bHY6yh-d38247/cod-bxxqqq_bHY6yh-d38247.xml (hunk 3 above)
Comment (GrantSchreib) [reply bHY6yh-d38247/cod-bxxqqq_bHY6yh-d38247.xml]: fixed it

**Review by jeffreycwitt — APPROVED** [review]

**Pull request merged**